### PR TITLE
Issue 153

### PR DESCRIPTION
**Pull request by cranfillkd**
Connects issue #153

### CODE REVIEW

**Comment by cranfillkd** [comment]
This PR had several conflicts. I fixed them but let me know if they cause trouble during the review.

File: src/app/utility/help-panel/help-panel.component.ts [hunk 1]
```diff
@@ -10,23 +10,33 @@ import { IdbFacility } from 'src/app/models/idb';
...
+    ) { 
+    router.events.subscribe((route) => {
+      if (route instanceof NavigationEnd) {
+        this.helpText = route.url.replace('/utility/','');
+        this.selectedSource = this.helpText.split('energy-consumption/')[1];
+      }
+    });
+  }
 
   ngOnInit() {
```
Comment (rmroot) [inline src/app/utility/help-panel/help-panel.component.ts:30]: I'm guessing this was a merge issue. We need the selected facility and save functions for the site to source section

File: src/app/utility/help-panel/help-panel.component.ts [hunk 2]
```diff
@@ -10,23 +10,33 @@ import { IdbFacility } from 'src/app/models/idb';
...
+  selectedSource: string;
   selectedFacilitySub: Subscription;
-  constructor(private facilityDbService: FacilitydbService) { }
+  selectedFacilityName: string = 'Facility';
+  
+  constructor(
+    private router: Router,
+    private facilityDbService: FacilitydbService,
+    ) { 
+    router.events.subscribe((route) => {
```
Comment (rmroot) [inline src/app/utility/help-panel/help-panel.component.ts:22]: The help ends up blank right away if you don't navigate to the component. (on refresh)

in ngOnInit() the same logic can be used with this.router.snapshot.url (it doesn't update on events so will still need this). I'd create a function that takes the url and sets these two properties.

File: src/app/utility/help-panel/help-panel.component.html [hunk 3]
```diff
@@ -16,15 +16,43 @@ <h4>Site or Source Energy?</h4>
...
+    <div *ngIf="helpText === 'predictors'">
+        <p>These are independent variables that will be used for <strong>{{ selectedFacilityName }}'s</strong> regression analysis. Predictors can be used to explain the change in your utility use.</p>
+        <p>Use “Add/Edit Predictors” to Add new Predictors for your analysis.</p>
+        <p>Use “Add Predictor Entry” to input your monthly predictor data.</p>
+        <p>Use “Import Data” to bulk import Predictor data in csv or excel format.</p>
+    </div>
+    
+    <div *ngIf="helpText === 'upload-data'" class="mt-5">
+        <h4>Template Downloads</h4>
+        <ul>
```
Comment (rmroot) [inline src/app/utility/help-panel/help-panel.component.html:51]: Relative paths don't work in electron application but we don't use these .csv templates anymore. There's just the one xlsx template now

**Review by rmroot — CHANGES_REQUESTED** [review]

**Review by rmroot — APPROVED** [review]

**Pull request merged**